source/leuven/tier2_hardware/kuleuven_storage.rst

```diff
@@ -12,18 +12,18 @@ The storage is organized according to the :ref:`VSC storage guidelines<data loca
 +--------------------------+--------+----------+--------+----------------+
 |$VSC_DATA                 | NFS    |  VSC     |YES     | 75 GB          |
 +--------------------------+--------+----------+--------+----------------+
-|$VSC_SCRATCH              | Lustre |  genius  |NO      | 500 GB         |
-|$VSC_SCRATCH_SITE         |        |          |        |                |
+|$VSC_SCRATCH              | Lustre |  Genius  |NO      | 500 GB         |
+|$VSC_SCRATCH_SITE         |        |  & wICE  |        |                |
 +--------------------------+--------+----------+--------+----------------+
-|$VSC_SCRATCH_NODE         | ext4   | genius,  |NO      | 200 GB         |
-|                          |        | job only |        |                |
+|$VSC_SCRATCH_NODE         | ext4   |  Genius  |NO      | 200 GB         |
```

Comment:
The size of `$VSC_SCRATCH_NODE` depends on the cluster, on wICE it is 600GB I belief.

Reply:
fixed via `8c4d458`